lib/matplotlib/offsetbox.py

```diff
@@ -357,7 +362,7 @@ def draw(self, renderer):
         to the given *renderer*.
         """
         w, h, xdescent, ydescent, offsets = self.get_extent_offsets(renderer)
-        px, py = self.get_offset(w, h, xdescent, ydescent, renderer)
```

Comment:
Any user who has used `set_offset` with a callable will be broken by this (as they no longer get the inputs they expect).